pkg/ui/ui.go

```diff
@@ -58,20 +39,19 @@ func NewBaseUI(logger log.Logger, menuTmpl string, funcMap template.FuncMap, tmp
 func (bu *BaseUI) serveStaticAsset(w http.ResponseWriter, req *http.Request) {
 	fp := route.Param(req.Context(), "filepath")
 	fp = filepath.Join("pkg/ui/static", fp)
-	bu.serveAsset(fp, w, req)
+	if err := bu.serveAsset(fp, w, req); err != nil {
+		level.Warn(bu.logger).Log("msg", "Could not get file", "err", err, "file", fp)
+		w.WriteHeader(http.StatusNotFound)
+	}
```

Comment:
:+1:

Reply:
I made all the suggested changes, is there anything else i need to change?